backend/src/schema/types/schema.gql

```diff
@@ -4,16 +4,15 @@ type Query {
   currentUser: User
   # Get the latest Network Statistics
   statistics: Statistics!
-  findPosts(query: String!, limit: Int = 10): [Post]!
+  findPosts(filter: String!, limit: Int = 10): [Post]!
     @cypher(
       statement: """
-      CALL db.index.fulltext.queryNodes('full_text_search', $query)
+      CALL db.index.fulltext.queryNodes('full_text_search', $filter)
       YIELD node as post, score
       MATCH (post)<-[:WROTE]-(user:User)
       WHERE score >= 0.2
       AND NOT user.deleted = true AND NOT user.disabled = true
       AND NOT post.deleted = true AND NOT post.disabled = true
-      AND NOT user.id in COALESCE($filter.author_not.id_in, [])
```

Comment:
If you remove this hack (it is a hack) what is there as a replacement?

Is it possible that the cypress test checking the search results is a false negative? The build server should not pass...

Reply:
no @roschaefer, I think what there is as a replacement is your function that gets called here https://github.com/Human-Connection/Human-Connection/blob/master/backend/src/schema/resolvers/posts.js#L35